data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+				goto elevatorend
+			label entertaining
+			`	"Glad to hear that, I am," She says. "A rare occasion it is, that to an outsider of our Coalition, our games we can show."`
+			label elevatorend
+			`	As the elevator finishes its descent and the doors begin to open, the Heliarch catches your attention once more. "Hope that you enjoy this next event, I do very much, though the best at it in my time I was not," she says as she begins guiding the audience along a path leading to the other stadium in the valley. Fortunately, it is much shorter than the track through the forest, so it isn't long before the escorted crowd arrives and begins filling the seats inside. In the center of this stadium, you see a rectangular court of some sort with small mounds dotted around the field and six large holes arranged evenly around the sides. Tall nets stretching to the ceiling separate the field and the seating area in a way you find to be slightly worrying.`
+			`	Once the audience is seated, an announcer again comes on the loudspeaker, reiterating the contestants that have qualified, though in an order that seems to be different than their ranking in the last event. Sensing your confusion, a Kimek sitting next to you with a translation device explains that the winning contestants from the previous event are resting. In the meantime they have been divided into six competing teams, each assigned a different color and with total scores as close to equal as possible. Apparently each participant will still be scored individually at the end of the event based on their specific contributions, in addition to their team's overall performance. As you sit waiting for the event to start, you are reminded of how out of place your are among the alien beings, and you realize you are quite likely the first human to ever witness these games.`
+			`	A slightly awkward half hour passes in which the uniqueness of your presence is more and more noted, before the thirty Saryds enter the arena and regain the crowd's attention. Holding what looks like fancy polo mallets, the members of each team are once more introduced to the crowd with a drone fly over, and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`
+			`	Once the game starts, you start to realize that it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defences. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`
```

Comment:
```suggestion
			`	Once the game starts, you start to realize that it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`
```